specification/billing/resource-manager/Microsoft.Billing/preview/2018-08-01-preview/billing.json

```diff
@@ -3099,6 +3190,35 @@
                 }
             }
         },
+        "Policy": {
+            "description": "The Policy.",
+            "type": "object",
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                }
+            ],
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/PolicyProperties",
+                    "title": "Policy properties"
+                }
+            }
+        },
+        "PolicyProperties": {
+            "description": "The properties of policy.",
+            "properties": {
+                "reservationPurchasesAllowed": {
+                    "description": "The reservationPurchasesAllowed flag.",
+                    "type": "boolean"
+                },
+                "marketplacePurchasesAllowed": {
+                    "description": "The marketplacePurchasesAllowed flag.",
+                    "type": "boolean"
```

Comment:
See comments above on the boolean

Reply:
There will not be any value other than true or false.